games/naval_battle.c

```diff
@@ -0,0 +1,802 @@
+/**
+ * @file
+ * @author [Carlos Rafael](https://github.com/CarlosZoft)
+ * @author [Herick Lima](https://github.com/hericklima22)
+ * @brief [naval_battle](https://en.wikipedia.org/wiki/Battleship_(game))
+ * implementation in C using only the stdio.h library.
+ * @details  Naval battle is a game, to be played by two people. It consists of
+ * knocking down the enemy ship, through shots , when hit the ship is
+ * revealed with the respective number of its size. Example: size 3 = 3 3 3 on
+ * the board.
+ */
+#include <stdio.h>
```

Comment:
Please add a brief description of what the library/header is being used for (see the example below).

```c++
#include <iostream>    /// for IO operations
#include <vector>     /// for std::vector
```